framework/src/org/checkerframework/common/basetype/BaseTypeVisitor.java

```diff
@@ -2919,20 +2936,10 @@ public Void visitIdentifier(IdentifierTree node, Void p) {
     }
 
     protected void checkAccess(IdentifierTree node, Void p) {
-        // Do not add functionality to this method body.
-        // Add it in the method called below instead.
-        checkAccess(node, p, true);
-    }
-
-    protected void checkAccess(IdentifierTree node, Void p, boolean doCheckPreconditions) {
         MemberSelectTree memberSel = enclosingMemberSelect();
         ExpressionTree tree;
         Element elem;
 
-        if (doCheckPreconditions) {
-            checkPreconditions(node, TreeUtils.elementFromUse(node), false);
```

Comment:
Why can you remove all these invocations of checkPreconditions?
Or are you removing changes that you had added in an earlier iteration?